### PR TITLE
rocm-4.1.0 release-add git repo path, modify hipfft package recipe

### PR DESCRIPTION
**Pull request by srekolam**

### CODE REVIEW

**Comment by haampie** [comment]
depends_on(..., when='@master')?

File: var/spack/repos/builtin/packages/hipfft/package.py [hunk 1]
```diff
@@ -8,20 +8,32 @@
...
+    version('master', branch='master')
     version('4.1.0', sha256='885ffd4813f2c271150f1b8b386f0af775b38fc82b96ce6fd94eb4ba0c0180be')
 
-    depends_on('hip@4.1.0')
-    depends_on('hip-rocclr@4.1.0')
-    depends_on('rocfft@4.1.0')
-    depends_on('rocm-cmake@4.1.0', type='build')
+    depends_on('cmake@3:', type='build')
+
+    for ver in ['4.1.0']:
```
Comment (haampie) [inline var/spack/repos/builtin/packages/hipfft/package.py:28]:
```suggestion
    for ver in ['4.1.0', 'master']:
```

**Review by haampie — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/hipfft/package.py (hunk 1 above)
Comment (srekolam) [reply var/spack/repos/builtin/packages/hipfft/package.py:28]: @haampie , i have not tested the master for rocfft etc.. hence did not want to add it at this moment. its a work in progress.

**Comment by srekolam** [comment]
@haampie , can you please approve this PR.

**Review by haampie — APPROVED** [review]

**Pull request merged**